admin/app/views/workarea/admin/pricing_skus/index.html.haml

```diff
@@ -46,6 +46,7 @@
                 = check_box_tag 'select_all', nil, false, id: 'select_all', class: 'checkbox__input', data: { bulk_action_select_all: '' }
                 = label_tag 'select_all', t('workarea.admin.bulk_actions.select_all'), class: 'checkbox__label'
             %th= t('workarea.admin.fields.sku')
+            = append_partials('admin.additional_fields_labels')
```

Comment:
So, the append names are globally scoped - in this case "additional fields labels" could mean anywhere! Can you use a more specific name like `admin.pricing_skus_additional_field_labels`?

Reply:
Agree on this one completely. Will work on implementing the change here.